text/0000-clarify-api-namespace.md

```diff
@@ -0,0 +1,59 @@
+- Start Date: 2015-08-10
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+Clarify API description namespace and Iron out issues which rose during API Blueprint serialization to API description namespace.
+
+# Motivation
+The purpose of this document is to clear inconsistencies in the format and make it easier to build tools around it.
+
+# Detailed design
+
+## HREF Variables
+We need to clarify about the value of the `content` in the `hrefVariables` element.
+
+## Data Structure
+We don't need a `Data Structure` because an element which is an `object`, `extend` or `array` can be considered as a data structure element.
+
+*(The current spec is a lot confusing. What is the `Data Structure Element`? Why is `Data Structure` inheriting from it?)*
+
+## Resource
+Since `hrefVariables` is an element, we should move it to be under `content` rather than leave it under `attributes`.
+
+## Transition
+Similarily to the Resource, we need to move `hrefVariables` to be under `content`. `Data Structure` in Resource is under `content`, but it is under `attributes` in Transition. To resolve this inconsistency, we need to move `Data Structure` to be under `content`. Another reason for it is that `Data Structure` is an element.
+
+## Transition Example
+Value of `content` should be changed to `[]` instead of `null`.
+
+## Category
+We need to clarify what `transitions` category is about. If it is being reserved for future use, we need to add reserved keyword.
```

Comment:
I think this was introduced by @kylef or @smizell in the call for being able to define transitions at the root level. I am up for any clarification if needed, but frankly I think it is not. It is just a group of transitions. It does not have a syntax in API Blueprint and neither scenarios have. 

I am :+1: to **REMOVE** the "reserved" word from "scenario - Reserved. Category is set of steps." since it is not really reserved. It can be used as a category semantics. Just because something does not have a counter part in API Blueprint syntax it does not mean it has to be reserved here. (Simply the conversion will ignore it)